src/lib.rs

```diff
@@ -1582,6 +1583,10 @@ pub mod __rt {
     use alloc::boxed::Box;
     use alloc::rc::Rc;
 
+    pub mod marker {
+        pub use super::super::marker::*;
+    }
```

Comment:
```suggestion
    pub use super::marker;
```
Right?

Reply:
That was my first thought too, but...

```
error[E0365]: `marker` is only public within the crate, and cannot be re-exported outside
    --> src\lib.rs:1593:13
     |
1593 |     pub use super::marker;
     |             ^^^^^^^^^^^^^ re-export of crate public `marker`
     |
     = note: consider declaring type or module `marker` with `pub`
```

The issue is that `pub use super::marker;` requires the original `marker` module to be public, but this would mean that people can import it via `wasm_bindgen::marker` as well (AFAIK), which we don't want.

So I created a new module of the same name and re-exported only the public elements inside the original `marker` module.

Reply:
Okay lets do this actually ... just move `__rt` to its own file.
```rust
#[path = "rt/mod.rs"]
pub mod __rt;
```
Then we can just split it up there and you can use `pub mod marker`.